src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -139,8 +139,7 @@ implication        ,X86   ,AVX512VBMI_VL        ,AVX512VBMI
 implication        ,X86   ,AVX512VBMI_VL        ,AVX512BW_VL
 implication        ,X86   ,AVX10v1              ,AVX2
 implication        ,X86   ,AVX10v1              ,FMA
-implication        ,X86   ,AVX10v1_V256         ,AVX10v1
-implication        ,X86   ,AVX10v1_V512         ,AVX10v1_V256
+implication        ,X86   ,AVX10v1_V512         ,AVX10v1
```

Comment:
I think we're missing:
```
implication ,X86 ,AVX10v1 ,EVEX
```
and
```
implication ,X86 ,AVX512F ,EVEX
```

Reply:
and then `EVEX` should be the thing that is:
```
implication        ,X86   ,EVEX                 ,AVX2
implication        ,X86   ,EVEX                 ,FMA
```

Reply:
Implications added. makes sense.